src/swift/flwr/Sources/Flower/Common/Typing.swift

```diff
@@ -88,6 +162,7 @@ public struct Status: Equatable {
     }
 }
 
+/// Parameters request for a client
```

Comment:
This is not a request:
```suggestion
/// Parameters message.
```